html-tables/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Tables</title>
+</head>
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <thead>
+      <tr>
+        <th>Student Name</th>
+        <th>Course</th>
+        <th>Grade</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Grace Hopper</td>
+        <td>Computer Science</td>
+        <td>100</td>
+      </tr>
+    </tbody>
+    <tbody>
```

Comment:
Whoops! You only need one `tbody`, it can hold all the `tr` elements for the body of the table.